docs/admin/logs/_index.md

```diff
@@ -0,0 +1,176 @@
+
+![System diagram: forwarders and co-located collector and nginx](system.svg)
+
+<!-- yuml.me UML rendering of:
+[Forwarder1]logs->[Collector]
+[Forwarder2]logs->[Collector]
+
+// Add notes
+[Collector]->[shared volume]
+[nginx]-[shared volume]
+-->
+
+The [`fluent-bit-collector.yaml`](./fluent-bit-collector.yaml) defines a
+StatefulSet, as well as a Kubernetes Service which allows accessing and reading
+the logs from within the cluster. The supplied configuration will create the
+monitoring configuration in a namespace called `logging`.
+
+You can apply the configuration by entering the command:
+
+```shell
+kubectl apply --filename https://github.com/knative/docs/raw/main/docs/install/collecting-logs/fluent-bit-collector.yaml
+```
+
+The default configuration will classify logs into:
+
+- Knative apps, or pods with an `app=Knative` label
+- Non-Knative apps
+
+The logs default to logging with the pod name; this can be changed by updating the `log-collector-config` ConfigMap before or after installation. Once the ConfigMap is updated, you must restart Fluent Bit. You can do this by deleting the pod and letting the StatefulSet recreate it.
+
+To access the logs through your web browser, enter the command:
+
+```shell
+kubectl port-forward --namespace logging service/log-collector 8080:80
+```
+
+And then visit http://localhost:8080/.
+
+You can also open a shell in the `nginx` pod and search the logs using unix
+tools, by entering the command:
+
+```shell
+kubectl exec --namespace logging --stdin --tty --container nginx log-collector-0
+```
+
+## Setting up the forwarders
+
+Follow the [Fluent Bit directions for installing on Kubernetes](https://docs.fluentbit.io/manual/installation/kubernetes) documentation to set up a Fluent Bit DaemonSet which forwards logs to ElasticSearch by default.
+
+When the directions call for creating the ConfigMap, you need to replace the ElasticSearch configuration with the [`fluent-bit-configmap.yaml`](./fluent-bit-configmap.yaml), or add the following block to the ConfigMap, and update the
+`@INCLUDE output-elasticsearch.conf` to be `@INCLUDE output-forward.conf`.
+
+```yaml
+output-forward.conf: |
+  [OUTPUT]
+      Name            forward
+      Host            log-collector.logging
+      Port            24224
+      Require_ack_response  True
+```
+
+## Local collector
+
+**NOTE:** This describes a development environment setup, and is not appropriate
+for production.
+
+If you are using a local Kubernetes cluster for development, you can create a `hostPath` PersistentVolume to store the logs on your desktop operating system. This allows you to use your usual desktop tools on the files without needing Kubernetes-specific tools.
+
+The PersistentVolumeClaim will look something like this, but the `hostPath` will
+vary based on your Kubernetes software and host operating system. Some example
+values are documented below.
+
+```yaml
```

Comment:
The options below this would be a great candidate for content tabs in mkDocs as well! 

These are rendered: 

```
=== "Kind"

        Some text

=== "Docker Desktop

        Some Text
```

And would render like this: 
<img width="698" alt="Screen Shot 2021-05-05 at 10 36 45 AM" src="https://user-images.githubusercontent.com/16281246/117158890-cf02e700-ad8d-11eb-95a3-64e41126a2ba.png">

Reply:
Good - put it in a guide please 😄

Reply:
yes yes just wanted to get you thinking about it :)